roborock/code_mappings.py

```diff
@@ -335,8 +337,9 @@ class RoborockMopIntensityQRevoMaster(RoborockMopIntensityCode):
     low = 201
     medium = 202
     high = 203
+    custom = 204
     custom_water_flow = 207
-    custom = 209  # SmartPlan
+    smart_mode = 209
```

Comment:
I don’t think I have a translation key for smart mode so I’m not sure if this will work

Reply:
I guess I’m not opposed to it, but you’ll need to add it to the translations if you do a dependency bump

Reply:
I based it off of this existingblock, which uses the same code:

```
class RoborockMopIntensityS8MaxVUltra(RoborockMopIntensityCode):
    off = 200
    low = 201
    medium = 202
    high = 203
    custom = 204
    max = 208
    smart_mode = 209
    custom_water_flow = 207
```

I guess it could already have been missing a translation.

Reply:
Oh looks like I do have it and I forgot

Reply:
Looks like it's there in the integration, if that's what you mean: https://github.com/home-assistant/core/blob/afae257a129b2b2d50b0448b9923a436717f47d6/homeassistant/components/roborock/strings.json#L348